app/src/main/res/layout/dialog_progress.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:gravity="center"
+    android:padding="16dp">
+
+
+
+    <ImageView
+    android:id="@+id/loading_gif"
+    android:layout_width="100dp"
+    android:layout_height="100dp" />
+
+    <TextView
+        android:id="@+id/progress_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:text="Initialising face recognition. Please wait..."
```

Comment:
if we add this text in string file we can see this text in user specific langauge